# Baseline V1

Team rating:

\[
\text{rating} = \text{wins} - \text{losses}
\]

For each historical tournament game, the higher-rated team is predicted to win. Rating ties receive half credit.

In [1]:
from pathlib import Path
import pandas as pd

import sys

PROJECT_ROOT = Path.cwd().parents[1]
sys.path.insert(0, str(PROJECT_ROOT)) if str(PROJECT_ROOT) not in sys.path else None

from src.pipelines.baseline_v1 import run_all_seasons, summarize_results

ModuleNotFoundError: No module named 'src.baselines.predictor'

In [ ]:
results = run_all_seasons()
overall = summarize_results(results)

pd.set_option("display.max_rows", 15)

display(
    results.assign(
        Accuracy=(results["Accuracy"] * 100).round(2)))

,Season,Games,Correct,Incorrect,Ties,Accuracy
0,1985,63,41,21,1,65.87
1,1986,63,39,23,1,62.70
2,1987,63,42,18,3,69.05
3,1988,63,35,28,0,55.56
4,1989,63,38,20,5,64.29
...,...,...,...,...,...,...
35,2021,66,37,28,1,56.82
36,2022,67,37,29,1,55.97
37,2023,67,31,33,3,48.51
38,2024,67,36,29,2,55.22


In [7]:
print(f"Games: {int(overall['Games'])}")
print(f"Correct: {int(overall['Correct'])}")
print(f"Incorrect: {int(overall['Incorrect'])}")
print(f"Ties: {int(overall['Ties'])}")
print(f"Effective accuracy: {overall['Accuracy']:.2%}")

Games: 2585
Correct: 1554
Incorrect: 914
Ties: 117
Effective accuracy: 62.38%


## Result

The first baseline achieves approximately **62.38% effective accuracy** across historical NCAA tournament games.

This serves as the reference point for future rating methods that incorporate factors such as opponent strength, margin of victory, location, and recency.